css/src/alerts.css

```diff
@@ -0,0 +1,43 @@
+.alert {
+	display: flex;
+	align-items: baseline;
+	position: relative;
+	padding: 16px;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+	font-size: 14px;
+	font-weight: 400;
+	line-height: 1.5;
+	margin: 16px 0;
+}
+
+.alert__error {
```

Comment:
Please use `--` instead of `__` and with a `yoast` prefix as @IreneStr suggested. 
For example: `.yoast-alert--error`